v8/minimals/conf.py.example

```diff
@@ -0,0 +1,131 @@
+import time
+
+BLOG_AUTHOR = "your web blog author here.."  # (translatable)
```

Comment:
The `conf.py.example` file should be as minimal as possible.

Reply:
Oh Ok, i will fix it and push again later.
Thanks for your inform